style-dictionary/themes/next/tokens/color/base.json

```
@@ -2,335 +2,180 @@
-          "code": { "value": "orange-600" },
-          "classname": { "value": "t-sage--color-orange-600" }
-        },
-        "700": {
-          "hex": { "value": "#b3501e" },
-          "code": { "value": "orange-700" },
-          "classname": { "value": "t-sage--color-orange-700" }
-        },
-        "800": {
-          "hex": { "value": "#5a260c" },
-          "code": { "value": "orange-800" },
-          "classname": { "value": "t-sage--color-orange-800" }
-        },
-        "900": {
-          "hex": { "value": "#5a260c" },
-          "code": { "value": "orange-900" },
-          "classname": { "value": "t-sage--color-orange-900" }
         }
       },
       "purple": {
-        "50": {
-          "hex": { "value": "#eee4f6" },
-          "code": { "value": "purple-50" },
-          "classname": { "value": "t-sage--color-purple-50" }
-        },
         "100": {
           "hex": { "value": "#eee4f6" },
           "code": { "value": "purple-100" },
           "classname": { "value": "t-sage--color-purple-100" }
         },
         "200": {
-          "hex": { "value": "#eee4f6" },
+          "hex": { "value": "#d9c2ef" },
           "code": { "value": "purple-200" },
           "classname": { "value": "t-sage--color-purple-200" }
         },
         "300": {
-          "hex": { "value": "#d9c2ef" },
+          "hex": { "value": "#8e5ad8" },
           "code": { "value": "purple-300" },
           "classname": { "value": "t-sage--color-purple-300" }
         },
         "400": {
-          "hex": { "value": "#d9c2ef" },
+          "hex": { "value": "#50348a" },
           "code": { "value": "purple-400" },
           "classname": { "value": "t-sage--color-purple-400" }
         },
         "500": {
-          "hex": { "value": "#8e5ad8" },
+          "hex": { "value": "#381c5e" },
           "code": { "value": "purple-500" },
           "classname": { "value": "t-sage--color-purple-500" }
-        },
-        "600": {
-          "hex": { "value": "#8e5ad8" },
-          "code": { "value": "purple-600" },
-          "classname": { "value": "t-sage--color-purple-600" }
-        },
-        "700": {
-          "hex": { "value": "#50348a" },
-          "code": { "value": "purple-700" },
-          "classname": { "value": "t-sage--color-purple-700" }
-        },
-        "800": {
-          "hex": { "value": "#381c5e" },
-          "code": { "value": "purple-800" },
-          "classname": { "value": "t-sage--color-purple-800" }
-        },
-        "900": {
-          "hex": { "value": "#381c5e" },
-          "code": { "value": "purple-900" },
-          "classname": { "value": "t-sage--color-purple-900" }
         }
       },
       "charcoal": {
-        "50": {
-          "hex": { "value": "#f9fafa" },
-          "code": { "value": "charcoal-50" },
-          "classname": { "value": "t-sage--color-charcoal-50" }
-        },
         "100": {
-          "hex": { "value": "#8d939a" },
+          "hex": { "value": "#8D939A" },
```

The linter is looking for lower case hex values. This should clear up the check failure
```suggestion
          "hex": { "value": "#8d939a" },
```